File: app/src/main/java/xyz/volgoak/wordlearning/data/LinkDao.java
package xyz.volgoak.wordlearning.data;

import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Insert;
import android.arch.persistence.room.Update;

import xyz.volgoak.wordlearning.entities.Link;

/**
 * Created by alex on 1/5/18.
 */

@Dao
public interface LinkDao {

    @Insert
    void insertLinks(Link...links);

}

File: data/src/main/java/com/attiladroid/data/db/Dao/LinkDao.kt
package com.attiladroid.data.db.Dao

import android.arch.persistence.room.Dao
import android.arch.persistence.room.Insert

import com.attiladroid.data.entities.Link

/**
 * Created by alex on 1/5/18.
 */

@Dao
interface LinkDao {

    @Insert
    fun insertLinks(vararg links: Link)

}
